spec/components/support_interface/audit_trail_component_spec.rb

```diff
@@ -76,4 +76,23 @@ def render_result
 
     expect(render_result.text).to include('Permission relationship between training provider A and ratifying provider B changed')
   end
+
+  context 'pagination' do
+    before do
+      stub_const('SupportInterface::AuditTrailComponent::PAGY_PER_PAGE', 2)
+
+      travel_temporarily_to(Time.zone.local(2019, 10, 1, 12, 10, 0)) do
```

Comment:
Is there a reason we need to travel to 2019? I worry about having these set dates in the tests. Would it work to just travel anytime in the past, eg `1.day.ago`, or does it have to be 2019?

Reply:
The date isn't important, it's the same date as is used throughout the file

Reply:
I think it would be good to move away from using explicit dates unless we are testing something very specific to do with the dates. I have come across quite a few things related to end of cycle where it looks like tests only pass because we are testing things for a date range that will never happen in real life (ie, before 2023). It doesn't seem like 2019 is particularly sinister here, but just good practice to use a generalisation unless the specific date is important to the test.